requirements-latest.txt

```diff
@@ -0,0 +1,45 @@
+# Non RobotPy packages, some are required by RobotPy because for simplicity we don't pin them,
+# just explicitly list that they are required
```

Comment:
The comments in this file probably need to be updated to reflect this file's purpose.

Reply:
Simplified that file to just the robotpy packages and updated the comments, hopefully its more clear now